subprojects/language-native/src/main/java/org/gradle/language/swift/tasks/CreateBundle.java

```diff
@@ -14,40 +14,36 @@
  * limitations under the License.
  */
 
-package org.gradle.nativeplatform.test.xctest.tasks;
+package org.gradle.language.swift.tasks;
 
 import org.gradle.api.Action;
 import org.gradle.api.DefaultTask;
 import org.gradle.api.Incubating;
 import org.gradle.api.file.CopySpec;
-import org.gradle.api.file.Directory;
 import org.gradle.api.file.DirectoryVar;
-import org.gradle.api.file.RegularFile;
 import org.gradle.api.file.RegularFileVar;
-import org.gradle.api.provider.Provider;
 import org.gradle.api.tasks.InputFile;
 import org.gradle.api.tasks.OutputDirectory;
 import org.gradle.api.tasks.TaskAction;
-import org.gradle.nativeplatform.test.xctest.internal.SwiftStdlibToolLocator;
+import org.gradle.language.swift.internal.SwiftStdlibToolLocator;
 import org.gradle.process.ExecSpec;
 
 import javax.inject.Inject;
-import java.io.File;
 
 /**
- * Creates XCTest bundle for execution.
+ * Creates Apple bundle.
  *
- * @since 4.2
+ * @since 4.3
  */
 @Incubating
-public class CreateXcTestBundle extends DefaultTask {
+public class CreateBundle extends DefaultTask {
     private final RegularFileVar informationFile;
     private final RegularFileVar executableFile;
     private final DirectoryVar outputDir;
     private final SwiftStdlibToolLocator swiftStdlibToolLocator;
 
     @Inject
-    public CreateXcTestBundle(SwiftStdlibToolLocator swiftStdlibToolLocator) {
+    public CreateBundle(SwiftStdlibToolLocator swiftStdlibToolLocator) {
```

Comment:
doesn't this specifically create a _Swift_ bundle, rather than _a_ bundle of any kind?

Reply:
yes, I would call the task class something like `CreateSwiftBundle` to make this clearer until we make it more general.

Reply:
Good point, done.